docs/cloud-adoption/appendix/azure-scaffold.md

```diff
@@ -2,247 +2,331 @@
-> 
-> 
+In real life, scaffolding is used to create the basis of a structure. The scaffold guides the general outline and provides anchor points for more permanent systems to be mounted. An enterprise scaffold is the same: a set of flexible controls and Azure capabilities that provide structure to the environment, and anchors for services built on the public cloud. It provides the builders (IT and business groups) a foundation to create and attach new services keeping speed of delivery in mind.
 
-The following image describes the components of the scaffold. The foundation relies on a solid plan for departments, accounts, and subscriptions. The pillars consist of Resource Manager policies and strong naming standards. The rest of the scaffold comes from core Azure capabilities and features that enable a secure and manageable environment.
+The scaffold is based on practices we have gathered from many engagements with clients of various sizes. Those clients range from small organizations developing solutions in the cloud to large multi-national enterprises and independent software vendors who are migrating workloads and developing cloud-native solutions. The enterprise scaffold is "purpose-built" to be flexible to support both traditional IT workloads and agile workloads; such as, developers creating software-as-a-service (SaaS) applications based on Azure platform capabilities.
 
-![scaffold components](./_images/components.png)
+The enterprise scaffold is intended to be the foundation of each new subscription within Azure. It enables administrators to ensure workloads meet the minimum governance requirements of an organization without preventing business groups and developers from quickly meeting their own goals. Our experience shows that this greatly speeds, rather than impedes, public cloud growth.
 
 > [!NOTE]
-> Azure has grown rapidly since its introduction in 2008. This growth required Microsoft engineering teams to rethink their approach for managing and deploying services. The Azure Resource Manager model was introduced in 2014 and replaces the classic deployment model. Resource Manager enables organizations to more easily deploy, organize, and control Azure resources. Resource Manager includes parallelization when creating resources for faster deployment of complex, interdependent solutions. It also includes granular access control, and the ability to tag resources with metadata. Microsoft recommends that you create all resources through the Resource Manager model. The enterprise scaffold is explicitly designed for the Resource Manager model.
-> 
-> 
+> Microsoft has released into preview a new capability called Azure Blueprint that will enable you to package, manage, and deploy common images, templates, policies, and scripts across subscriptions and management groups. This capability is the bridge between the scaffold's purpose as reference model and deploying that model to your organization.
```

Comment:
Do you have a link for Azure Blueprint?

Reply:
I don't.  The content will be released next week and I was going to do an update then, also for Azure Resource Graph

Reply:
Does this mean that we need to hold on publication until the Azure Blueprint link is released? Or do we publish now & have @Petertaylor9999 update to add the link in a few days?

Reply:
I think so? Otherwise we're referencing something that doesn't exist yet.

Reply:
I'm OK with waiting until the link is released. It should be any day so timing it to release with that content makes sense.

Reply:
Just looked and it appears that the BluePrints documentation has been released. I'm updating the content right now.

Reply:
Thanks @rdendtler This change avoids potential confusion & allows us to meet quality standards.